PhysicsTools/NanoAOD/python/globals_cff.py

```diff
@@ -70,5 +71,17 @@
     ),
 )
 
+tauSpinnerTable = tauSpinnerTableProducer.clone(
+    src = 'prunedGenParticles',
+    name = 'TauSpinner',
+    theta = [0, 0.25, 0.5, -0.25, 0.375],
+    pdfSet = 'NNPDF31_nnlo_hessian_pdfas',
+    cmsE = 13600,
+    defaultWeight = 1
+)
+(~run3_common).toModify(
+    tauSpinnerTable, cmsE = 13000
+)
+
 globalTablesTask = cms.Task(beamSpotTable, rhoTable)
-globalTablesMCTask = cms.Task(puTable,genTable,genFilterTable)
+globalTablesMCTask = cms.Task(puTable,genTable,genFilterTable,tauSpinnerTable)
```

Comment:
I think it's better to add this to `genparticles_cff.py` instead of `globals_cff`.

Reply:
TauSpinner weights are global event weights similar to weights for PDF set replicas, etc. But if you think that `genparticles_cff.py` is better place for it I can move it.

Reply:
In that case it probably fits better in `genWeightsTable_cfi.py`.

Reply:
OK, I'll move it there.

Reply:
After checking some thinking I found the the following:
*  the `genWeightsTable_cfi.py` defines only one module (it is `cfi` file), namely `genWeightsTable` producer, and adding other module into this `cfi` will be misleading (also because of the name of the file);
* one can prepare a separate `cfi` file for `tauSpinnerTable` producer (similar to the `genWeightsTable_cfi.py`) or keep the producer where it is, i.e. in `globals_cff.py`, as the TauSpinner weights are global in similar sense as generator weights stored by `genTable` producer. I prefer the latter option, but I will prepare a new `cfi` if it is found better.

Reply:
Finally, I decided to create a new dedicated `cfi` file for `tauSpinnerTable` producer.